mapillary/controller/image.py

```diff
@@ -85,87 +90,126 @@ def get_image_close_to_controller(
     )
 
     # Filtering for the attributes obtained above
-    if unfiltered_data["features"] != {} and unfiltered_data["features"][0]["properties"] != {}:
-        filtered_data.extend(
-            pipeline(
-                data=unfiltered_data,
-                components=[
-                    # Filter using kwargs.min_date
-                    {"filter": "min_date", "min_timestamp": kwargs["min_date"]}
-                    if "min_date" in kwargs
-                    else {},
-                    # Filter using kwargs.max_date
-                    {"filter": "max_date", "min_timestamp": kwargs["max_date"]}
-                    if "max_date" in kwargs
-                    else {},                
-                    # Filter using kwargs.image_type
-                    {"filter": "image_type", "tile": kwargs["image_type"]}
-                    if "image_type" in kwargs
-                    else {},
-                    # Filter using kwargs.organization_id
-                    {"filter": "organization_id", "organization_ids": kwargs["org_id"]}
-                    if "org_id" in kwargs
-                    else {},
-                    # Filter using kwargs.radius
-                    {"filter": "haversine_dist", "radius": kwargs["radius"],
-                        "coords": [longitude, latitude]}
-                    if "radius" in kwargs
-                    else {},
-                ],
-            )
+    if (
+        unfiltered_data["features"] != {}
```

Comment:
features should be a list if empty and not an object, right? [ ] instead of { } -- see like on geojson.io

Reply:
Yes, you're right, must've missed it. Changed in latest commit.